test/performance/ferguson/remote-record-write-1.graph

```diff
@@ -0,0 +1,5 @@
+perfkeys: PUTs:, PUTs:
+files: remote-record-write-1-c.dat, remote-record-write-1-c-cache.dat
+graphkeys: c PUTs, c-cache PUTs
+ylabel: Count
+graphtitle: remote-record-write-1
```

Comment:
Is there a way we could better distinguish tests with and without the "-1" suffix? It seems like it could be confusing when discussing/searching graphs.